library/src/scripts/components/headers/pieces/VanillaMobileHomeHeader.tsx

```diff
@@ -46,14 +48,24 @@ export class VanillaMobileHomeHeader extends React.Component<IProps> {
         const headerClasses = vanillaHeaderClasses();
         const classes = vanillaHeaderHomeClasses();
 
+        const themeLogo = getLogo(LogoType.MOBILE);
+        let logoData = dummyLogoData;
```

Comment:
I see this is actually duplicated. It might be best to wrap it up a redux connected `<ThemeLogo />` component and share it. It can use redux's connect function to map in the data.